include/ros_control_boilerplate/generic_hw_interface.h

```diff
@@ -80,9 +80,23 @@ class GenericHWInterface : public hardware_interface::RobotHW
   /** \brief Read the state from the robot hardware. */
   virtual void read(ros::Duration &elapsed_time) = 0;
 
+  /** \brief Delegate RobotHW::read() calls to read() */
+  virtual void read(const ros::Time& /*time*/, const ros::Duration& period)
+  {
+    ros::Duration elapsed_time = period;
+    read(elapsed_time);
+  }
+
   /** \brief Write the command to the robot hardware. */
   virtual void write(ros::Duration &elapsed_time) = 0;
 
+  /** \brief Delegate RobotHW::write() calls to write() */
```

Comment:
Please expand the comment to cover the arguments here too

Reply:
Done in revision